src/cmd/cli/command/commands.go

```diff
@@ -984,13 +986,36 @@ var deploymentsListCmd = &cobra.Command{
 	Args:        cobra.NoArgs,
 	Short:       "List deployments",
 	RunE: func(cmd *cobra.Command, args []string) error {
-		loader := configureLoader(cmd)
-		projectName, err := loader.LoadProjectName(cmd.Context())
+		historyOnly, err := cmd.Flags().GetBool("history")
+		if err != nil {
+			return err
+		}
+
+		limit, err := cmd.Flags().GetUint32("limit")
+		if err != nil {
+			return err
+		}
```

Comment:
I wouldn't bother; we don't do this anywhere else
```suggestion
		historyOnly, _ := cmd.Flags().GetBool("history")
		limit, _ := cmd.Flags().GetUint32("limit")
```